templates/scripts/pre-env.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Maybe we should do something similar to what I did in post-apply, and source a script that is specific to sendgrid, then we can optionally include it. We may want to make this support optional via a parameter.